Security Access support on alert + No more lawyer-traitors

MilenVolf opened the pull request:
## О запросе слияния
1. Служба безопасности впредь будет получать все доступы не только в `Красный` код, но и в коды `Гамма` и `Дельта`.
2. Также Адвокат более не сможет получить полный доступ в описанные выше коды, так как фильтр по доступу сменился с `Бриг` на `СлужбаБезопасности`.

## Почему / Баланс
1. Во время критических событий, когда устанавливается Гамма код и при запуске Ядерное боеголовки в код Дельта, сотрудники службы безопасности были вынуждены просить доступ/открыть двери главой/сотрудником отдела, теряя драгоценное время, необходимое для устранения угрозы.
2. Игроки привыкли брать Адвоката не с целью самой роли - помогать заключённым при вынесении несправедливых сроков и заключений, а с целью выбивания Предателя на этой роли и вместе с этим получением полного доступа во все отделы, что, делало из Адвоката ультимативного Предателя (или идеальную добычу для Предателей).

**Чейнджлог**

:cl:
- tweak: После нескольких поправок в блюспейс-ретрансляторе, члены отдела Службы Безопасности отныне также получают доступ в отделы в коды Гамма и Дельта.
- tweak: Адвокаты больше не получают доступов в отделы в Красный, Гамма и Дельта коды.

MilenVolf committed:
BS access fix

MilenVolf committed:
Security Access support on alert + No more lawyer-traitors

Security Access support on alert gamma&delta levels + No more lawyer-trators

MilenVolf commented:
Буду ожидать комментариев по поводу пункта `2.` по большей части.

AruMoon commented:
Адвокат изначально против СБ. Он не работает с ними и у него не должно быть арсенала СБ, а значит и доступы ему не нужны

AruMoon commented:
А убрать ему антага не против

MilenVolf committed:
Lawyers can be traitors again

After some discussion on discord, I decided to leave the opportunity for lawyer to stab securitys'back, when they're least aware
